### DISCUSSION TITLE
is it possible to change weather globally (to snow for example)?

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#1817] [Q&A] [answered]
there is only one option currently present in global that is `Force Thunder`.

![image](https://github.com/YimMenu/YimMenu/assets/139460769/7336aca0-8a7d-4da4-95a0-2cc31f172f61)

In the past the feature was undone in this pr #494.
@xiaoxiao921?

**Comment by maybegreat48** [comment, marked as answer]
> is it possible to change weather globally (to snow for example)?

No
